### PR TITLE
feat: beta version of the (web-)importer

### PR DESCRIPTION
**Pull request by kptdobe**
I would like to use helix-website to host the first beta version of (web-)importer.
While this is still a prototype, I think it would be useful to see if someone really can use it to digest html to create docx.
Similar architecture than the Sidekick. 
The source code is located here: https://github.com/kptdobe/helix-web-importer (still temporary location). This PR would then be automated on release (manual for now).
If approved, I'll create a page to document how to use.

### CODE REVIEW

File: tools/importer/module.js [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+      ? parent.insertBefore(createTag(config), before)
+      : parent.appendChild(createTag(config)));
+  }
+
+   /**
+   * Returns the sidekick configuration.
+   * @private
+   * @param {sidekickConfig} cfg The sidekick config (defaults to {@link window.hlx.sidekickConfig})
+   * @param {Location} location The current location
+   * @returns {Object} The sidekick configuration
```
Comment (rofe) [inline tools/importer/module.js:142]: jsdoc still mentions sidekick 😉

File: tools/importer/module.js [hunk 2]
```diff
@@ -0,0 +1,397 @@
...
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+/* eslint-disable no-console, no-alert */
+
+'use strict';
+
+(() => {
```
Comment (rofe) [inline tools/importer/module.js:16]: I know it's the same in the sidekick module, but I just realized that a IIFE is not needed in JS modules. Unless you want to run async code outside of a function.

File: tools/importer/app.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      return;
+    }
+
+    const host = appScript.getAttribute('src').replace('/app.js', '').split('?')[0];
+    const moduleScript = document.createElement('script');
+    moduleScript.id = 'hlx-importer-module';
+    moduleScript.src = `${host}/module.js`;
+    const config = JSON.parse(appScript.dataset['config'] || "{}");
+    moduleScript.addEventListener('load', () => {
+      window.hlx.initImporter(Object.assign({
```
Comment (rofe) [inline tools/importer/app.js:32]: Didn't `eslint` flag this? This should use the spread operator instead of `Object.assign`.

**Review by rofe — APPROVED** [review]

**Comment by kptdobe** [comment]
I have applied linting on the source project to clean up the code.

**Pull request merged**